roles/serf/files/serf.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Serf
-After=auditd.service systemd-user-sessions.service time-sync.target
+After=network.target auditd.service systemd-user-sessions.service time-sync.target
```

Comment:
@mapuri makes sense.. Let me try it out standalone first, without `network.target` dependency to see if it works the way it should. and will do both the changes in this PR itself.